app/javascript/react/screens/App/Overview/components/ModalWizard/ModalWizard.js

```diff
@@ -14,10 +16,23 @@ const ModalWizard = props => {
     activeStep,
     numSteps,
     goToStep,
-    children
+    children,
+    formContainer
   } = props;
   const onFirstStep = activeStepIndex === 0;
   const onFinalStep = activeStepIndex === numSteps - 1;
+
+  const formHasErrors = () =>
+    find(formContainer, form => form.syncErrors) !== undefined;
```

Comment:
Do we need to use lodash `find()` when we can use `Array.prototype.find()` now with babel? I'm fine with either, just noticing this. We use lodash enough elsewhere that it doesn't matter much.

Reply:
Let's keep `lodash`